### DISCUSSION TITLE
The block reward is appearing as 0

### DISCUSSION THREAD

**Discussion by LenChoi** [blockscout/blockscout#10451] [Q&A]
I have modified the Clique algorithm in Geth to issue block rewards during block validation. However, the reward amount is appearing as 0. Could you advise on the settings required to ensure that the reward amount is displayed correctly? Thank you.
<img width="324" alt="스크린샷 2024-07-19 오후 12 01 14" src="https://github.com/user-attachments/assets/bf7d7d4a-1f03-41c6-947e-44275809ca13">

**Comment by avenbreaks** [comment]
it's part of your custom blockchain, unfortunately Clique PoA is no longer supported by their team.
